src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { todosAction } from './features/todos';
+import { currentTodoActions } from './features/currentTodo';
+import { filterActions } from './features/filter';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [openTodo, setOpenTodo] = useState<boolean>(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isOpenTodo, setIsOpenTodo] = useState<boolean>(false);
```